[2024-05-04T23:34:35.490+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data scheduled__2024-05-03T23:34:19.339583+00:00 [queued]>
[2024-05-04T23:34:35.497+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data scheduled__2024-05-03T23:34:19.339583+00:00 [queued]>
[2024-05-04T23:34:35.497+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-04T23:34:35.497+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-05-04T23:34:35.498+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-04T23:34:35.505+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): transform_data> on 2024-05-03 23:34:19.339583+00:00
[2024-05-04T23:34:35.509+0000] {standard_task_runner.py:55} INFO - Started process 62 to run task
[2024-05-04T23:34:35.513+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'covid_analytics_dag', 'transform_data', 'scheduled__2024-05-03T23:34:19.339583+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/covid_analytics.py', '--cfg-path', '/tmp/tmp0obnw6as']
[2024-05-04T23:34:35.513+0000] {standard_task_runner.py:83} INFO - Job 101: Subtask transform_data
[2024-05-04T23:34:35.561+0000] {task_command.py:376} INFO - Running <TaskInstance: covid_analytics_dag.transform_data scheduled__2024-05-03T23:34:19.339583+00:00 [running]> on host 1287e4bf5d77
[2024-05-04T23:34:35.608+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Diego Munera
AIRFLOW_CTX_DAG_ID=covid_analytics_dag
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-03T23:34:19.339583+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-03T23:34:19.339583+00:00
[2024-05-04T23:34:35.617+0000] {credentials.py:1180} INFO - Found credentials in environment variables.
[2024-05-04T23:34:44.356+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-05-04T23:34:44.381+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=covid_analytics_dag, task_id=transform_data, execution_date=20240503T233419, start_date=20240504T233435, end_date=20240504T233444
[2024-05-04T23:34:44.442+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-04T23:34:44.459+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
